packages/prerender-data-provider/package.json

```diff
@@ -1,29 +1,24 @@
 {
   "name": "@preact/prerender-data-provider",
   "version": "3.0.1",
-  "description": "Prerender Prerender data provider for preact-cli apps",
+  "description": "Prerender data provider for preact-cli apps",
   "keywords": [
     "preact",
     "preact-cli"
   ],
-  "author": "Preact Authors <core@preactjs.com>",
-  "homepage": "https://github.com/developit/preact-cli#readme",
-  "license": "MIT",
-  "main": "src/index.js",
-  "directories": {
```

Comment:
It doesn't seem like this serves much purpose?

https://docs.npmjs.com/cli/v8/configuring-npm/package-json#directories